bbc/simorgh
# WSTEAM1-789 - `liveBlogPosting` SEO schema

Resolves https://jira.dev.bbc.co.uk/browse/WSTEAM1-789

Overall changes
======
- Adds `liveBlogPosting` Schema metadata for SEO. This extracts the `paragraph` block types from each post (ignoring other block types for now) to construct a `blogPosting` object that is used to list the posts on the current page in the Schema SEO metadata
- Adds `tsconfig` to Next.js Cypress to allow proper type checking of global Cypress functions

Code changes
======
- Adds `getLiveBlogPosting` function to construct Schema SEO metadata for each post on the current page
- Adds tests around this function
- Adds basic e2e for checking existence of SEO metadata on the Live page

Testing
======
1. Visit https://www.test.bbc.com/pidgin/live/c7p765ynk9qt
2. Open the Schema metadata using an extension such as DetailedSEO
3. Check that the `liveBlogPosting` field exists
4. Check that each of the posts is represented within the `liveBlogPosting` section

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
